src/modules/props.ts

```diff
@@ -48,10 +51,26 @@ function updateProps(formerVNode: ElementVNode, vNode: ElementVNode): void {
   props = props || {}
 
   for (const key in formerProps)
-    if (PROPERTIES_TO_SKIP.indexOf(key) === -1 && !props[key])
+    if (!PROPERTIES_TO_SKIP[key] && !props[key]) {
```

Comment:
Nested if-statements are considered poor coding style and lead to convoluted and hard-to-understand code. Below is my suggestion on how to fix them. Consider using other names for the variables if you find it so.

```
const propertyIsMissing = !PROPERTIES_TO_SKIP[key] && !props[key]
const keyIsClassName = propertyIsMissing && key === 'className'
const shouldRemoveAttribute = propertyIsMissing && ATTRIBUTE_TO_REMOVE[key]

if (keyIsClassName)
  removePreviousClassName(formerProps[key], element)

if (shouldRemoveAttribute)
  element.removeAttribute(key)

if (propertyIsMissing)
  delete element[key]
```